README.md

```diff
@@ -82,7 +83,7 @@ apt update && apt install git-lfs
 git lfs install
 
 # Clone repo and update submodules.
-git clone <repo link>
+git clone <repo link> #ENSURE IT IS THE HTTP LINK NOT SSH#
```

Comment:
Can you add a note about this above? Something along the lines of "Make sure to clone the repo using HTTPS (not SSH) since using Git in side the development container currently doesn't work if cloned via SSH"